plugins/dynamic-home-page/app-config.janus-idp.yaml

```diff
@@ -0,0 +1,7 @@
+# please keep this in sync with packages/app/src/App.tsx
+dynamicPlugins:
+  frontend:
+    janus-idp.backstage-plugin-dynamic-home-page:
```

Comment:
```suggestion
    internal.backstage-plugin-dynamic-home-page:
```

Reply:
I switched to janus-idp everywhere so that we don't need to use an `@internal` prefix for this feature (in the docs). I expect that we extract this plugin in 1.4.